cpu/native/periph/flash.c

```diff
@@ -24,172 +27,76 @@
 #include <unistd.h>
 #include <stdio.h>
 #include <err.h>
-#include <errno.h>
 #include <fcntl.h>
 #include <stdio.h>
-#include <sys/stat.h>
 #include <sys/types.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
 
 #include "cpu-conf.h"
 #include "periph/flash.h"
 
 #include "native_internal.h"
 
-#define ENABLE_DEBUG (1)
+#define ENABLE_DEBUG (0)
 #include "debug.h"
 
-#define FLASH_ERROR_SIMULATION (1)
```

Comment:
I assumed as much, but I suggest to keep it out for now. It has no benefit unless there is an implementation using it.